Gems/MotionMatching/JupyterNotebooks/LearnedMotionMatching.ipynb

```diff
@@ -0,0 +1,141 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "a45e3d25",
+   "metadata": {},
+   "source": [
+    "# Load data helpers"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25a44238",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "def PrintGreen(text):\n",
+    "    print('\\x1b[6;30;42m' + text + '\\x1b[0m')\n",
+    "    \n",
+    "def PrintRed(text):\n",
+    "    print('\\33[41m' + text + '\\x1b[0m')\n",
+    "\n",
+    "def LoadData(filename, rowsName, columnsName):\n",
+    "    newDataframe = pd.read_csv(filename, na_values = 'null')\n",
+    "    if newDataframe.shape[0] > 0 and newDataframe.shape[1] > 0:\n",
+    "        PrintGreen(\"Loading \" + filename + \" succeeded\");\n",
+    "    else:\n",
+    "        PrintRed(\"Loading \" + filename + \" failed!\");\n",
+    "\n",
+    "    print(rowsName + \" = \" + str(newDataframe.shape[0]))\n",
+    "    print(columnsName + \" = \" + str(newDataframe.shape[1]))\n",
+    "\n",
+    "    return newDataframe\n",
+    "\n",
+    "# Ensure to show all columns\n",
+    "pd.set_option('display.max_columns', None)"
+   ]
+  },
```

Comment:
Interesting. We were just discussing pros and cons of storing some shader code samples inside of JSON for material canvas nodes. Was this entered directly in the JSON or from another tool that did not require escaping?

Reply:
That is a Jupyter Notebook (https://jupyter.org/), so not a custom O3DE file format. Several editors can handle that file format like VSCode or PyCharm.